modules/shop/productDetails.tsx

```diff
@@ -221,20 +229,20 @@ export default function ProductDetails() {
     'Offers tutorials to set up and customize the template.',
   ];
 
-  const readMoreBtn = document.querySelector('.read-more-btn') as HTMLElement | null;
-  const text = document.querySelector('.text-wrapper') as HTMLElement | null;
-
-  const readMore = () => {
-    text?.classList.toggle('line-clamp-3');
-    text?.classList.toggle('line-clamp-none');
-    if (readMoreBtn?.textContent === 'Read more') {
-      readMoreBtn.textContent = 'Read less';
-      console.log('it reads more');
-    } else if (readMoreBtn?.textContent === 'Read less') {
-      readMoreBtn.textContent = 'Read more';
-      console.log('it reads less');
-    }
-  };
+  // const readMoreBtn = document.querySelector('.read-more-btn') as HTMLElement | null;
+  // const text = document.querySelector('.text-wrapper') as HTMLElement | null;
+
+  // const readMore = () => {
+  //   text?.classList.toggle('line-clamp-3');
+  //   text?.classList.toggle('line-clamp-none');
+  //   if (readMoreBtn?.textContent === 'Read more') {
+  //     readMoreBtn.textContent = 'Read less';
+  //     console.log('it reads more');
+  //   } else if (readMoreBtn?.textContent === 'Read less') {
+  //     readMoreBtn.textContent = 'Read more';
+  //     console.log('it reads less');
+  //   }
+  // };
```

Comment:
Remove unused code.

Reply:
Okay that iI didn't do 😅. I'll do that now